e2e-tests/insurance/cypress/e2e/journeys/application-submission/multiple-policy-type/policy-conditions/submit-an-application-multiple-policy-type-maximum-cover.spec.js

```diff
@@ -0,0 +1,29 @@
+import { APPLICATION } from '../../../../../../../constants';
+
+context(
+  'Insurance - submit an application - Multiple policy type, no broker - As an Exporter, I want to submit my completed credit insurance application with a value greater than 500000, So that UKEF can process and make a decision on my application',
```

Comment:
should we add without-broker to name of file?

Reply:
Instead i've removed it from the test description for both single and multiple "maximum cover" tests. It doesn't make sense to say "no broker" as there are many other conditions not mentioned. Would be overkill to list them all